radicle-httpd/src/api/v1/projects.rs

```diff
@@ -378,6 +386,98 @@ async fn issues_handler(
     Ok::<_, Error>(Json(issues))
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+pub struct IssueCreation {
+    pub title: String,
+    pub description: String,
+    pub tags: Vec<Tag>,
+}
+
+// // Create a new issue
+// // `POST /projects/:project/issues`
+async fn issue_creation_handler(
```

Comment:
```suggestion
async fn issue_create_handler(
```

Reply:
This has been removed, since I want to focus in this PR on authorize a session.